tests/radv/test_radv_ipv6_ra.py

```diff
@@ -24,10 +29,16 @@
 
 """
 
-@pytest.fixture(scope="module")
-def radv_test_setup(duthosts, rand_one_dut_hostname, ptfhost, tbinfo):
-    duthost = duthosts[rand_one_dut_hostname]
+@pytest.fixture(scope="module", autouse=True)
+def radv_test_setup(request, duthosts, ptfhost, tbinfo):
+    if 'dualtor' in tbinfo['topo']['name']:
```

Comment:
We can avoid this entire if/else block by using the fixture `rand_selected_dut` as our device for testing, and we can use the fixture `toggle_all_simulator_ports_to_rand_selected_dut` instead of `toggle_all_simulator_ports_to_upper_tor`